### ISSUE TITLE
Is it possible to have PC automatically put in sleep mode when TV is turn off (use power button on the remote control) just like the Ps5?

### ISSUE THREAD

**Issue by WilliamLeGod** [JPersson77/LGTVCompanion#36]
I was wondering if Is it possible to have PC automatically put in sleep mode when TV is turn off (use power button on the remote control) just like the Ps5?

**Comment by JPersson77** [comment]
I will consider this feature request, even though it's a bit incompatible with the scope of the application (to work like a normal PC-monitor)

**Comment by sonnettie** [comment]
yeah or even shutdown? honestly don't know how you'd do this though over ip though? unless you have an addon lg webos app? or maybe a phone home ping to the tv ip address every mins or so? anyway thanks for your work i couldn't believe in this day and age where TV's are used mainstream with pc gaming this isn't implemented at hardware level on 30x and 40x GPUs its not the 1990s TV's have vrr, gsync, game mode, all, freesync, etc and GPUs have hdmi, hdmi 2.1 etc so why not cec? i appreciate you coding this its exactly what everyone that wants to game in the living room with a 83inch tv on a sofa instead of in there basement on a 24 inch monitor lol, im sure ive heard your name "JPerssson77" did you do work on the lg root via web code? im actually running that right now lol to get add free youtube with sponsor skip etc
